crates/ursa-utils/src/shutdown.rs

```diff
@@ -0,0 +1,45 @@
+//! The shutdown module makes it easy for us to have one source for sending shutdowns
```

Comment:
It might be useful to also add a handler for SIGTERM. What do you think?

Reply:
Yes! that's a great suggestion. I added SIGTERM and SIGQUIT handlers now.